feat: Add dlc theme to SearchInput

lux-v opened the pull request:
**Component Created:** SearchInput.vue

**Stories:** ~/stories/SearchInput.stories.js

**Spec:** ~/stories/SearchInput.spec.js

**Notes:**

<img width="1171" height="873" alt="image" src="https://github.com/user-attachments/assets/43c9cf3d-ba3c-40de-9b9c-d7fd631c00f1" />


**Checklist:**

-   [ ] I checked that it is working locally in the dev server
-   [x] I checked that it is working locally in the storybook
-   [ ] I checked that it is working locally in the 
library-website-nuxt dev server
-   [x] I added a screenshot of it working
-   [x] UX has reviewed and approved this
-   [ ] I assigned this PR to someone on the dev team to review
-   [x] I used a conventional commit message
-   [x] I assigned myself to this PR

pghorpade commented:
@lux-v   we may not need this component as we have[ Search Input](https://github.com/UCLALibrary/ucla-library-website-components/blob/main/packages/vue-component-library/src/lib-components/SearchInput.vue) component used in the following components

https://ucla-library-storybook.netlify.app/?path=/story/search-search-home--default
https://ucla-library-storybook.netlify.app/?path=/story/nav-search--default-with-placeholder-prop
https://ucla-library-storybook.netlify.app/?path=/story/search-generic--default

lux-v commented:
> @lux-v we may not need this component as we have[ Search Input](https://github.com/UCLALibrary/ucla-library-website-components/blob/main/packages/vue-component-library/src/lib-components/SearchInput.vue) component used in the following components
> 
> https://ucla-library-storybook.netlify.app/?path=/story/search-search-home--default https://ucla-library-storybook.netlify.app/?path=/story/nav-search--default-with-placeholder-prop https://ucla-library-storybook.netlify.app/?path=/story/search-generic--default

@pghorpade Ok, I removed `EntryField` component, and added "DLC" theme to `SearchInput` component.

pghorpade commented on packages/vue-component-library/src/lib-components/SearchInput.vue:
```diff
@@ -1,293 +1,127 @@
+  return props.clearIcon && !!props.modelValue && props.modelValue.length > 0
+})
+const classes = computed(() => [
+  'search-input-wrapper',
```
since you are using the class as search-input-wrapper, you need to use this in spec . This will fix cypress error on this story

pghorpade reviewed (approved):
👍

pghorpade commented:
:tada: This PR is included in version @ucla-library-monorepo/ucla-library-website-components-v1.34.0 :tada:

The release is available on [GitHub release](https://github.com/UCLALibrary/ucla-library-website-components/releases/tag/%40ucla-library-monorepo/ucla-library-website-components-v1.34.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
